api/envoy/extensions/transport_sockets/tls/v3/common.proto

```diff
@@ -253,7 +254,21 @@ message CertificateProviderPluginInstance {
   string certificate_name = 2;
 }
 
-// [#next-free-field: 15]
+// String matcher for subject alternative names, to match both type and value of the SAN.
+message SubjectAltNameMatcher {
+  enum SanType {
+    EMAIL = 0;
```

Comment:
Just to double check the intention here, is EMAIL the typical default value?
If not, then you should have an `UNSPECIFIED = 0` enum value (see [style guidelines](https://github.com/envoyproxy/envoy/blob/main/api/STYLE.md#api-style-guidelines)).

Reply:
Added default value, and test to verify that specifying it causes an error.